js/layout-javascript/news-feed-code.js

```diff
@@ -289,9 +308,13 @@ DynamicList.prototype.attachObservers = function() {
 
       _this.closeDetails();
     })
-    .on('click', '.list-search-icon .fa-sliders', function() {
+    .on('click keydown', '.list-search-icon .fa-sliders', function(event) {
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
       var $elementClicked = $(this);
       var $parentElement = $elementClicked.parents('.new-news-feed-list-container');
+      _this.$container.find('[data-filter-group]').prop('hidden', null);
```

Comment:
An empty line of spacing before and after each `var` block unless its in the beginning of a `{ }` block.

cc @upplabs-alex-levchenko @sofiiakvasnevska

```suggestion

      _this.$container.find('[data-filter-group]').prop('hidden', null);
```